app/controllers/waivers_controller.rb

```diff
@@ -6,11 +6,12 @@ def new
   end
 
   def create
-    if accepted_waiver?
-      sign_waiver
+    if !accepted_waiver?
```

Comment:
I think we should push this logic down into the interactor as well.  Maybe the signature for the interactor can be something like `SignWaiver.call(volunteer: current_volunteer, params: params)`?

Non-blocking